ansible/roles/master/tasks/main.yml

```diff
@@ -43,53 +37,32 @@
     controller_manager_token: "{{ tokens.results[0].content|b64decode }}"
     scheduler_token: "{{ tokens.results[1].content|b64decode }}"
     kubectl_token: "{{ tokens.results[2].content|b64decode }}"
+    kubelet_token: "{{ tokens.results[3].content|b64decode }}"
 
-- name: write the config file for the controller-manager
-  template: src=controller-manager.j2 dest={{ kube_config_dir }}/controller-manager
-  notify:
-    - restart controller-manager
+#TODO: add kube_apiserver_vip to defaults
```

Comment:
You should be able to remove this if you drop the var. When using the kube_apiserver_vip var, a default should not be set. To enable HA, an operator will define multiple [masters] inventory definitions and set the kube_apiserver_vip in group_vars/all.yml.